modules/gapi/src/executor/gapi_itt.hpp

```diff
@@ -0,0 +1,60 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+#ifndef OPENCV_GAPI_GAPI_ITT_HPP
```

Comment:
Should ITT be part of the `/executor`? maybe move it to `backends/common`?

Reply:
added a FIXME